pkg/cmd/server/start/start_allinone.go

```diff
@@ -99,6 +100,7 @@ func NewCommandStartAllInOne(basename string, out io.Writer) (*cobra.Command, *A
 	flags.StringVar(&options.NodeConfigFile, "node-config", "", "Location of the node configuration file to run from. When running from configuration files, all other command-line arguments are ignored.")
 	flags.BoolVar(&options.MasterOptions.CreateCertificates, "create-certs", true, "Indicates whether missing certs should be created.")
 	flags.BoolVar(&options.PrintIP, "print-ip", false, "Print the IP that would be used if no master IP is specified and exit.")
+	flags.StringVar(&options.ServiceNetworkCIDR, "portal-net", NewDefaultNetworkArgs().ServiceNetworkCIDR, "The CIDR string representing the network that portal/service IPs will be assigned from. This must not overlap with any IP ranges assigned to nodes for pods.")
```

Comment:
It used to get the --portal-net flag as part of its MasterArgs, but the earlier patch moved it from MasterArgs to NetworkArgs. My original version of this branch pulled the whole NetworkArgs into the all-in-one args, but you didn't want the extra options added to all-in-one, so I did this instead.

It *is* odd, but there's already a lot of oddness in the all-in-one arg handling because of how it's trying to combine the master and node args, while removing duplicates.